src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.ArrayList;
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.MESSAGE_LIST_SORTED;
+
+/**
+ * Sorts all the people in AddressBook by their names in alphabetically ascending order (case insensitive)
+ */
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the entire list of people in AddressBook by their names in alphabetically ascending order (case insensitive).\n"
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.